src/Bicep.LangServer.UnitTests/Handlers/BicepEditLinterRuleCommandHandlerTests.cs

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Bicep.Core.Text;
+using Bicep.Core.UnitTests;
+using Bicep.Core.UnitTests.Assertions;
+using Bicep.Core.UnitTests.Utils;
+using Bicep.LanguageServer.Handlers;
+using Bicep.LanguageServer.Telemetry;
+using FluentAssertions;
+using MediatR;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using OmniSharp.Extensions.JsonRpc;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Server;
+using Range = OmniSharp.Extensions.LanguageServer.Protocol.Models.Range;
+
+namespace Bicep.LangServer.UnitTests.Handlers
+{
+    [TestClass]
+    public class BicepEditLinterRuleCommandHandlerTests
+    {
+        [NotNull]
+        public TestContext? TestContext { get; set; }
+
+        private static readonly MockRepository Repository = new(MockBehavior.Strict);
+        private static readonly ISerializer Serializer = Repository.Create<ISerializer>().Object;
```

Comment:
We can get rid of MockRepository here by doing:
private static readonly ISerializer Serializer = StrickMock.Of<ISerializer>();